spyder/api/widgets/__init__.py

```diff
@@ -422,7 +425,7 @@ def get_actions(self, filter_actions=True):
             self.dock_action,
             self.undock_action,
             self.close_action,
-            'switch to ' + self._name,
+            self.toggle_view_action,
```

Comment:
Why is this change needed as part of this PR?

Reply:
Nothing specific to this PR, but it was something that needed fixing in the API